lib/path.js

```diff
@@ -854,6 +903,16 @@ const win32 = {
 
   format: FunctionPrototypeBind(_format, null, '\\'),
 
+  /**
+   * @param {string} path
+ * @returns {{
+ *  dir: string;
+ *  root: string;
+ *  base: string;
+ *  name: string;
+ *  ext: string;
+ *  }}
```

Comment:
```suggestion
  * @returns {{
  *  dir: string;
  *  root: string;
  *  base: string;
  *  name: string;
  *  ext: string;
  *  }}
```

I'm so sorry. Appearently, I miscounted the number of spaces before `*` in previous suggestion. IMO, it is better to align `*` in a column.